substrate/frame/revive/uapi/src/host/riscv64.rs

```diff
@@ -433,6 +434,10 @@ impl HostFn for HostFnImpl {
 		unsafe { sys::balance_of(address.as_ptr(), output.as_mut_ptr()) };
 	}
 
+	fn to_account_id(address: &[u8; 20], output: &mut [u8]) {
+		unsafe { sys::to_account_id(address.as_ptr(), output.as_mut_ptr()) }
+	}
```

Comment:
Has to be marked as unstable and moved down to the unstable APIs. In the runtime it is unstable by default. Unstable because it will be moved to a pre-compile as soon as they are available.

Reply:
Thanks for the comments, I just pushed the fixes!

>Unstable because it will be moved to a pre-compile as soon as they are available.

Could you elaborate on this? Will you remove the possibility to call these functions as host functions?

Reply:
Yes everything that is not a correspondence to an EVM opcode will move to a system precompile in order to be available to Solidity. In Solidity there is no way to call custom host functions so a pre-compile is the only way to make them available. We also don't want to maintain the same functionality with two different APIs.